hc-nested-list-behavior.html

```diff
@@ -107,7 +122,7 @@
      */
     itemOpenChanged(event, detail, idName = this.itemIdName, parts = this.parts) {
       // Block interactions while loading items
-      if (this.isLoading) return;
+      if (this.isLoading || !parts) return;
```

Comment:
Parts can't be undefined, this function is an event handler for `hc-nested-list-behavior#item-open-changed` which is fired by the node element inside of iron-list when it's rendered, which is rendered from _items that's built from parts.